### ISSUE TITLE
deeptools

### ISSUE THREAD

**Issue by chen5027** [ubccr/software-layer#196]
I recently started my lab at UB and found that an essential software called depptools is missing from the module collection so we cannot analyze our data.  This is a quote commonly used one, so could you install it?

Thanks,

Dahong

**Comment by tonykew** [comment]
Test build deepTools version 3.5.1 complete - more info to follow...

**Comment by tonykew** [comment]
deepTools version 3.5.1 has been published to the ccrsoft/2023.01 software release

```
vortex1$ module spider deeptools
----------------------------------------------------------------------------
  deeptools: deeptools/3.5.1
----------------------------------------------------------------------------
    Description:
      deepTools is a suite of python tools particularly developed for the
      efficient analysis of high-throughput sequencing data, such as
      ChIP-seq, RNA-seq or MNase-seq.


    You will need to load all module(s) on any one of the lines below before the "deeptools/3.5.1" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
vortex1$ module load gcc/11.2.0 openmpi/4.1.1 deeptools/3.5.1
vortex1$
```

deepTools includes multiple binaries:

```
vortex1$ ls ${EBROOTDEEPTOOLS}/bin
alignmentSieve     computeMatrix            multiBamSummary     plotHeatmap
bamCompare         computeMatrixOperations  multiBigwigSummary  plotPCA
bamCoverage        correctGCBias            plotCorrelation     plotProfile
bamPEFragmentSize  deeptools                plotCoverage
bigwigCompare      estimateReadFiltering    plotEnrichment
computeGCBias      estimateScaleFactor      plotFingerprint
vortex1$ 
```

e.g.

```
vortex1$ deeptools -h
usage: deeptools [-h] [--version]

deepTools is a suite of python tools particularly developed for the efficient analysis of
high-throughput sequencing data, such as ChIP-seq, RNA-seq or MNase-seq.
[...]
vortex1$ 
```
